src/ruby/lib/grpc/google_rpc_status_utils.rb

```diff
@@ -12,9 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-require_relative './grpc'
```

Comment:
nit: though perhaps redundant, `require structs` here, since this file uses that type

Reply:
Agreed. Added.